web/src/engine/websites/SeriManga.ts

```diff
@@ -0,0 +1,36 @@
+import { Tags } from '../Tags';
+import icon from './SeriManga.webp';
+import { Chapter, DecoratableMangaScraper, type Manga } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchCSS, FetchRequest } from '../FetchProvider';
+
+@Common.MangaCSS(/^https?:\/\/serimanga\.com\/manga\//, 'div.seri-img div.name')
+@Common.MangasMultiPageCSS('/mangalar?page={page}', 'ul.sub-manga-list li.mangas-item > a',1,1,0, Common.AnchorInfoExtractor(false, 'span.mlb-cat, span.mlb-star, span.wathced'))
```

Comment:
Is this a typo on the website or just here `span.wathced`?

Reply:
IIRC the website got a typo itself, but ill double check

Reply:
![image](https://user-images.githubusercontent.com/10671518/230644921-da289ea5-b2a6-46c8-b1b9-98e248ae4c64.png)